client/src/components/Input.js

```diff
@@ -51,3 +51,14 @@ export const Input = () => {
     </Form>
   );
 };
+
+export const ImageInput = styled.input`
```

Comment:
If this input is only meant to use for files, consider setting default props here:
https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values

You don't have to set `type="file" ` anymore when you call this component.